db/models/Snippet.js

```diff
@@ -0,0 +1,19 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+
+const snippetSchema = new Schema(
+  {
+    name: { type: String, required: true },
+    code: { type: String, required: true },
+    description: { type: String, required: false },
+    link: { type: String, required: false },
+    tag: { type: String, required: false },
```

Comment:
if those properties aren't required, you can omit the required key

```suggestion
    description: { type: String },
    link: { type: String },
    tag: { type: String },
```